backend/api/pipeline.proto

```diff
@@ -140,6 +140,13 @@ service PipelineService {
       get: "/apis/v1beta1/pipeline_versions/{version_id}/templates"
     };
   }
+
+  // Update the default pipeline version of a specific pipeline.
+  rpc UpdatePipelineDefaultVersion(UpdatePipelineDefaultVersionRequest) returns (google.protobuf.Empty) {
+    option (google.api.http) = {
+      post: "/apis/v1beta1/pipelines/{pipeline_id}/{version_id}"
```

Comment:
The endpoint seems a little ambiguous just reading from its structure
what do you think about?
```
/apis/v1beta1/pipelines/{pipeline_id}/default_version/{version_id}
```
or maybe
```
/apis/v1beta1/pipelines/{pipeline_id}/default_version
```
version id can be a query param

Reply:
I prefer: 

```
apis/v1beta1/pipelines/{pipeline_id}/default_version/{version_id}
```

will update and regenerate it API.